packages/ui-components/src/components/SelectOption/SelectOption.test.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as React from "react"
+import { cleanup, render, screen, waitFor } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import { Select } from "../Select/Select.component"
+import { SelectOption } from "../SelectOption/SelectOption.component"
+
+globalThis.console.warn = vi.fn()
+describe("SelectOption", () => {
+  afterEach(() => {
+    cleanup()
+    vi.clearAllMocks()
+  })
+
+  test("renders a SelectOption", async () => {
+    await waitFor(() =>
+      render(
+        <Select>
+          <SelectOption value="Option 1" />
+        </Select>
+      )
+    )
+    expect(console.warn).toHaveBeenCalledTimes(1)
+    const toggle = screen.getByRole("button")
+    expect(toggle).toBeInTheDocument()
+    await waitFor(() => userEvent.click(toggle))
```

Comment:
I understand that this is the migration so I'd not change it right now but something for the future that it is not recommended to use user interaction apis straight from  `userEvent` import we should rather create an instance `userEvent.setup()` and trigger events on that.
https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent

Reply:
But anyway I guess you need an act wrapper, I think that was a leftovers but good point I would add them to our upcoming ui-componetns epic here https://github.com/cloudoperators/juno/issues/570